One test failure on s390x

With pyRiemann 0.3 on the `s390x` architecture, one test fails:

```
=================================== FAILURES ===================================
____________ test_distance_func_separability[distance_wasserstein] _____________
dist = <function distance_wasserstein at 0x3fd8b4d4b80>
get_covmats = <function get_covmats.<locals>._gen_cov at 0x3fd7748d940>
    @pytest.mark.parametrize("dist", get_dist_func())
    def test_distance_func_separability(dist, get_covmats):
        n_matrices, n_channels = 1, 6
        covmats = get_covmats(n_matrices, n_channels)
>       assert dist(covmats[0], covmats[0]) == approx(0, abs=1e-7)
E       assert 1.2467205919833117e-07 == 0 ± 1.0e-07
E         comparison failed
E         Obtained: 1.2467205919833117e-07
E         Expected: 0 ± 1.0e-07
tests/test_utils_distance.py:99: AssertionError
```

It looks like the tolerance is slightly too tight to allow for floating-point implementation differences across architectures. I will follow up with a PR to loosen the `1e-7` in the assertion to at least `1.25e-7`—perhaps `2e-7` as a nice round number.

--- a/tests/test_utils_distance.py
+++ b/tests/test_utils_distance.py
@@ -96,7 +96,7 @@ def test_distance_func_geodesic(dist, get_covmats):
 def test_distance_func_separability(dist, get_covmats):
     n_matrices, n_channels = 1, 6
     covmats = get_covmats(n_matrices, n_channels)
-    assert dist(covmats[0], covmats[0]) == approx(0, abs=1e-7)
+    assert dist(covmats[0], covmats[0]) == approx(0, abs=2e-7)
 
 
 @pytest.mark.parametrize(

